src/amuse/test/suite/compile_tests/test_c_implementation_simplified.py

```diff
@@ -1,3 +1,4 @@
+import pytest
```

Comment:
I think its better to use the unittest skip

Reply:
it's best not to skip but sure :)